### PR TITLE
Fix some logging things.

### PR DESCRIPTION
**Pull request by fluffels**

### CODE REVIEW

File: jormungand.py [hunk 1]
```diff
@@ -99,7 +99,7 @@ def jormungand(json_config_file=None, plugin_roots=[], sources=[], logging=loggi
     parser.add_argument('--logfile', default=None)
     args = parser.parse_args()
     # Configure Logging
-    logging.basicConfig()
+    logging.basicConfig(level=args.loglevel)
     logger = logging.getLogger('JORMUNGAND')
     logger.setLevel(args.loglevel)
```
Comment (OOPMan) [inline jormungand.py]: @fluffels Shouldn't this be removed with the change to like 102?

Comment (fluffels) [inline jormungand.py]: 102?

Comment (OOPMan) [inline jormungand.py]: @fluffels On line 102 you set the general log level with the argument to basicConfig. On line 104, the JORMUNGAND logger gets the same log-level set. Shouldn't line 104 no longer be necessary?

**Comment by fluffels** [comment]
@OOPMan You are correct about the log level setting. I've updated the PR.

**Pull request merged**